wordpress-mobile/WordPress-iOS
# [SIWA] Show error alert when authentication fails

Ref #12442 
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/127

This updates the WPAuth pod to show the SIWA error alert.

To test:
- Go to either Log In or Sign Up.
- Select Continue with Apple.
- Cause the authentication to fail.
  - The easiest is to press the Home button while in the Apple flow.
- Verify an alert appears.

Update release notes:
